tests/integration/test_io.py

```diff
@@ -275,7 +275,7 @@ def test_append(self):
             self.assertIs(nwb.processing['test_proc_mod']['LFP'].electrical_series['test_es'].electrodes,
                           nwb.acquisition['timeseries2'].electrodes)
             errors = validate(io)
-            self.assertTrue(len(errors) == 0)
+            self.assertEqual(len(errors), 0, errors)
```

Comment:
Note: this change doesn't change any logic, but just displays what the validation errors are if any, making it easier to debug. (with the updated hdmf validator, this test fails due to the incompatible type for the filtering column of the electrodes table, but this test will currently still pass with the current dev version of hdmf.)

Reply:
Sounds good. I didn't know `assertEqual` had that functionality